SQL/New_patches/2018-08-27-GenderToSex.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `candidate` CHANGE COLUMN Gender Sex enum('Male','Female');
```

Comment:
@driusan I was about to write a comment that the patch was missing haha.

You are missing the probandsex change in the patch though

Reply:
good catch, added it